docker/Dockerfile.multiarch

```
@@ -0,0 +1,22 @@
+FROM --platform=$BUILDPLATFORM golang:1.16 AS build
+
+RUN apt-get install make
+WORKDIR /src
+COPY . .
+ARG TARGETOS TARGETARCH
+RUN --mount=type=cache,target=/root/.cache/go-build \
+    --mount=type=cache,target=/go/pkg \
+    make build
+
+FROM alpine:3.15
+RUN apk add --no-cache ca-certificates mailcap git openssh curl perl && \
```

Are `mailcap` and `perl` really needed.

```suggestion
RUN apk add --no-cache ca-certificates git openssh curl perl && \
```

---

These were also in the plugin-base image I replace.  This is now removed in the latest commit.